packages/fxa-shared/metrics/glean/web/index.ts

```diff
@@ -119,12 +119,12 @@ export const eventsMap = {
   },
 
   thirdPartyAuth: {
```

Comment:
I find it a bit confusing when the naming here doesn't follow the event category/name. Should these instead be sorted with `login` and `reg`?

Reply:
I will check the spreadsheet and have them consistent with that.  Thanks.

Reply:
@akkomar @ksiegler1 what do you think of renaming/moving the reg_ and login_ events here to the third_party_auth category?

Reply:
These events landed in prod ~a week ago so IMO it's still fine to rename them and not worry about loss of data if it makes taxonomy better organized.

Reply:
@ksiegler1 the latest revision of this patch renamed the third party auth apple/google reg/login start events.  I've updated the spreadsheet with the new names.  If there are already charts using the old names, we'll need to update them.  (If the patch makes into production as it is.)